wsgi.py

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/python
 import os
+from configapp.configapp import app as application
 
-virtenv = os.path.join(os.environ.get('OPENSHIFT_PYTHON_DIR','.'), 'virtenv')
+virtenv = os.path.join(os.environ.get('OPENSHIFT_PYTHON_DIR', '.'), 'virtenv')
 virtualenv = os.path.join(virtenv, 'bin/activate_this.py')
 try:
-    execfile(virtualenv, dict(__file__=virtualenv))
+    execfile(virtualenv, dict(__file__=virtualenv))  # noqa
 except IOError:
     pass
 #
 # IMPORTANT: Put any additional includes below this line.  If placed above this
 # line, it's possible required libraries won't be in your searchable path
 #
 
-from configapp.configapp import app as application
```

Comment:
@rstevanak why was this removed? It is used at [line 22](https://github.com/NaiveNeuron/PKSpace/pull/16/files#diff-ea8aaf2b76cffeafd45054ae9c1ff347R22) -- the whole thing will fail without it.